tools/src/test/resources/1.5/valid-vulnerability-1.5.json

```diff
@@ -80,19 +80,22 @@
           }
         ]
       },
-      "tools": [
-        {
-          "vendor": "Snyk",
-          "name": "Snyk CLI (Linux)",
-          "version": "1.729.0",
-          "hashes": [
-            {
-              "alg": "SHA-256",
-              "content": "2eaf8c62831a1658c95d41fdc683cd177c147733c64a93e59cb2362829e45b7d"
-            }
-          ]
-        }
-      ],
+      "tools": {
+        "components": [
+          {
+            "type": "application",
+            "group": "Snyk",
+            "name": "Snyk CLI (Linux)",
+            "version": "1.729.0",
+            "hashes": [
+              {
+                "alg": "SHA-256",
+                "content": "2eaf8c62831a1658c95d41fdc683cd177c147733c64a93e59cb2362829e45b7d"
+              }
+            ]
+          }
+        ]
```

Comment:
could you add `services`?
so we would have a complete example.

Reply:
Good catch. Added in https://github.com/CycloneDX/specification/pull/198/commits/bba0493f9c44faa67e436eee1fe0685c152ad6d7